src/components/graph/menus/network-modifications-menu.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import Menu from '@mui/material/Menu';
+import { useIntl } from 'react-intl';
+import { NestedMenuItem } from 'mui-nested-menu';
+import NodeMenuItem from './create-node-item';
+import MenuList from '@mui/material/MenuList';
+
+/**
+ * Menu to select network modification to create
+ * @param {Boolean} open Is the dialog open ?
+ * @param {EventListener} onClose Event to close the dialog
+ * @param onOpenDialog handle the opening of dialogs
+ * @param dialogs the list of dialog
+ * @param anchorEl anchorEl of fab Button
+ */
+const NetworkModificationsMenu = ({
+    open,
+    onClose,
+    onOpenDialog,
```

Comment:
see here, we have a `*Menu` component, why is there a dialog consideration here ? it should be renamed `onClick`, or `onItemClick` and the way this click open a dialog is independent of the menu itself

Reply:
new name: onClickItem